src/core/componentSystems/StyleSystem.js

```diff
@@ -13,7 +13,7 @@ export class StyleSystem extends MRSystem {
      * @description StyleSystem's default constructor with a starting framerate of 1/15.
      */
     constructor() {
-        super(false, 1 / 15);
+        super(false, 1 / 30);
```

Comment:
since this is updating, make sure also update the comment of line 13 for the new frame rate